pkg/component/networking/nodelocaldns/nodelocaldns.go

```diff
@@ -557,3 +566,29 @@ func (n *nodeLocalDNS) SetClusterDNS(dns []string) {
 func (n *nodeLocalDNS) SetDNSServers(servers []string) {
 	n.values.DNSServers = servers
 }
+
+func (n *nodeLocalDNS) SetIPFamilies(ipfamilies []gardencorev1beta1.IPFamily) {
+	n.values.IPFamilies = ipfamilies
+}
+
+func (n *nodeLocalDNS) GetIPVSAddress() (ipvsAddress string) {
```

Comment:
Why should this function be exported?

```suggestion
func (n *nodeLocalDNS) getIPVSAddress() (ipvsAddress string) {
```